src/aspire/basis/ffb_3d.py

```diff
@@ -267,12 +267,11 @@ def evaluate(self, v):
 
         pf = w_even + 1j * w_odd
         pf = m_reshape(pf, (n_theta * n_phi * n_r, n_data))
+        pf = np.moveaxis(pf, 0, -1)
```

Comment:
I suspect that some of these `moveaxis` and `transpose` can be removed, but this is not the time. Do we have a standing issue for this? Otherwise, could you create one?

Reply:
Nm I see #371 now.